care/emr/models/tag_config.py

```
@@ -0,0 +1,98 @@
+            self.parent_cache = [*self.parent.parent_cache, self.parent.id]
+            self.level_cache = self.parent.level_cache + 1
+            if self.parent.root_tag_config is None:
+                self.root_tag_config = self.parent
+            else:
+                self.root_tag_config = self.parent.root_tag_config
+            if not self.parent.has_children:
+                self.parent.has_children = True
+                self.parent.save(update_fields=["has_children"])
+        super().save()
+
+    def get_parent_json(self):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)